tests/integration/targets/dns_auth_nsg/tasks/main.yml

```diff
@@ -0,0 +1,170 @@
+---
+
+#TODO: add tests
+# The following require additional plugins to be supported.
+# - internal_secondaries
+# - nsgs
```

Comment:
What other plugins are required for the Nsg test ?

Reply:
adding host

Reply:
Nsg test case would just require the definition of a different NSG and no host

Reply:
added